### ISSUE TITLE
modifyvxlan.sh expect /var/run/cloud directory for LOCK, but directory require manual creation

### ISSUE THREAD

**Issue by levindecaro** [apache/cloudstack#4350]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
cloudstack-agent
~~~

##### CLOUDSTACK VERSION

~~~
4.14
~~~

##### CONFIGURATION

advanced networking, vxlan

##### OS / ENVIRONMENT

centos 7.8.2003

##### SUMMARY

Error when create guest instance, agent log show 

Sep 27 12:38:59 compute-12 java: WARN  [resource.wrapper.LibvirtPlugNicCommandWrapper] (agentRequest-Handler-3:) (logid:32fe0cb1)  Plug Nic failed due to com.cloud.exception.InternalErrorException: Failed to create vnet 2726: /usr/share/cloudstack-common/scripts/vm/network/vnet/modifyvxlan.sh: line 145: /var/run/cloud/vxlan.lock: No such file or directory

modifyvxlan.sh expect /var/run/cloud directory for persist LOCK file, but directory require manual creation

##### STEPS TO REPRODUCE

~~~
create zone with advanced networking without security group
import a centos/debian vm template
create a VPC 10.0.0.0/8
create a network zone 10.1.1.1/24
launch a vm instance
~~~

##### EXPECTED RESULTS

~~~
vm should be created
~~~

##### ACTUAL RESULTS

~~~
creation fail with "Command failed due to Internal Server Error"
~~~

**Comment by DaanHoogland** [comment]
fixed in #4373
can you confirm @levindecaro ?

**Comment by DaanHoogland** [comment]
verified by reported
